lib/Debugger.ProtocolHandler/RuntimeImpl.cpp

```diff
@@ -336,6 +377,13 @@ namespace JsDebug
 
                 if (GetTypeStringAndValue(arguments[i], &typeString, &objectValue))
                 {
+                    JsValueType valueType;
+                    JsGetValueType(objectValue, &valueType);
+                    if (valueType == JsValueType::JsObject) {
+                        JsValueRef error;
+                        objectValue = JsObjectToString(objectValue, &error);
```

Comment:
This can return nullptr if JsObjectToString() returns nullptr. In that case, should it fall back to calling GetTypeStringAndValue()? Otherwise the Value property is set to null but that is not correct.

Reply:
If revert back when JsObjectToString fails, then the user will [Object object] which is what we're trying to solve right.

Reply:
Given the code now, when JsObjectToString fails, what is the outcome? I believe it will return nullptr which isn't desirable, and having it fall back to "[Object object]" would be preferable.

Reply:
Alright